init/env_modules.py.in

```diff
@@ -0,0 +1,33 @@
+import os, sys, re, subprocess
```

Comment:
I would prefer to have the `python.py.in` moved in the repository as `env_modules.py.in` to avoid having 2 different files. The symlink will provide compatibility at installation time.

Reply:
I saw in the mailing list conversation that you noted the Windows artifacts haven't included python.py, so you're right that a symlink on the Linux installations makes more sense.